public/app/features/alerting/unified/state/actions.ts

```diff
@@ -385,17 +388,49 @@ export function deleteRuleAction(
   };
 }
 
+export const isRulerGrafanaRuleDTO = (rule: RulerRuleDTO): rule is RulerGrafanaRuleDTO => 'grafana_alert' in rule;
+
+// This method was introduced for making betterer happy (it was complaining about using as StoreState)
+const isStoreState = (something: unknown): something is StoreState => {
+  if (typeof something === 'object') {
+    return something ? 'unifiedAlerting' in something : false;
```

Comment:
I'm confused by this ternary operator condition. You're evaluating whether `something` is truthy or not, but it will always be truthy as you already know it's of type `object` at that point. I think you could simply `return 'unifiedAlerting' in something`.

Reply:
I'm checking that something it's not null